llvm/lib/MC/MCRepoTicketFile.cpp

```diff
@@ -319,11 +319,20 @@ llvm::mc::repo::realTicketDirectory(const StringRef &Path) {
   // Make absolute and remove component traversals, links, etc.
   llvm::SmallString<256> Out = Path;
   sys::path::remove_filename(Out);
+  llvm::SmallString<256> Result;
+  if (Out.size() == 0) {
+    // No path was specified: just use the cwd.
+    if (const std::error_code EC = llvm::sys::fs::current_path(Result)) {
+      return llvm::errorCodeToError(EC);
+    }
+    return Result;
+  }
+
   llvm::SmallString<256> Out2;
```

Comment:
The variable Out2 is not used, should it be removed?

Reply:
Yes. I've simplified the change.